managed/src/main/java/com/yugabyte/yw/common/KubernetesManager.java

```diff
@@ -35,6 +35,10 @@ public ShellResponse createNamespace(Map<String, String> config,
     return execCommand(config, commandList);
   }
 
```

Comment:
there's a few other functions in this file that take universePrefix as the arg. Let's rename it as namespace to avoid confusion.